app/controllers/users.js

```diff
@@ -60,3 +61,16 @@ exports.logIn = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.listUsers = (req, res, next) => {
+  const page = req.params.page;
+  const offset = limitOfUsersPerPage * (page - 1);
+
+  userService
+    .getAllUsersWithPagination(limitOfUsersPerPage, offset)
```

Comment:
again, for interacting with the db use either a service or methods in the model, not both.